onboarding/views.py

```diff
@@ -97,6 +101,22 @@ def profile_edit_view(request, userid):
             form.save()
             return redirect('profile', userid = userid)
     else:
-        form = CustomUserChangeForm(request.GET) 
-        
+       form = CustomUserChangeForm(initial={'first_name': user.first_name, 'last_name': user.last_name})
     return render(request, 'profile_edit.html', {'form': form})
+
+def change_password(request):
```

Comment:
this function is already provided from Django so you can change the password using this URL "http://127.0.0.1:8000/password_change/"